geth/transactions/types.go

```diff
@@ -68,3 +51,31 @@ func (args SendTxArgs) GetInput() hexutil.Bytes {
 func isNilOrEmpty(bytes hexutil.Bytes) bool {
 	return bytes == nil || len(bytes) == 0
 }
+
+func RpcCalltoSendTxArgs(args ...interface{}) SendTxArgs {
+	var err error
+	var fromAddr, toAddr common.Address
+
+	rpcCall := rpc.Call{Params: args}
+	fromAddr, err = rpcCall.ParseFromAddress()
+	if err != nil {
+		fromAddr = common.HexToAddress("0x0")
```

Comment:
Shouldn't we abort the transaction in the case of wrong/incorrect from/to addresses?

Reply:
that code is actually untouched by me. I'll take a look if I can make diffs smaller, because this shouldn't be removed/added.